source/quick-start.txt

```diff
@@ -37,11 +37,62 @@ to check your version of Node and NPM and installation instructions for your
 system, see `downloading and installing Node.js and npm
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
-Initialize the Project
-~~~~~~~~~~~~~~~~~~~~~~
-
 After you verify that you installed the required versions of Node and NPM,
-create a new project.
+create a new project by :ref:`cloning our js-starter repository
+<qs-download-the-repo>` or by :ref:`creating it manually <qs-manual-setup>`.
+
+.. _qs-download-the-repo:
+
+Clone the Project
+~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   This approach requires you to have `Git
+   <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+   installed.
+
+First, clone our `js-starter repository
+<https://github.com/mongodb-university/js-starter>`__  in your command
+line interface using the following command:
```

Comment:
```suggestion
<https://github.com/mongodb-university/js-starter>`__  in your shell using the following command:
```
Suggestion: I don't think "command line interface" is quite right here. A command line interface is how you use any command line program (I.E. git <some command> , <some command> is the CLI). I think this should be terminal, shell, REPL, etc..